src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -147,6 +147,13 @@ You can opt out of Android's Activity Instrumentation and App Start Instrumentat
 </manifest>
 ```
 
+### Slow and Frozen Frames
+
+Unresponsive UI, animation hitches, or just jank annoy users and degrade the user experience. Two measurements to track this unwanted experience are slow frames and frozen frames. A smoothly running app should try to avoid both. That's why the SDK adds these two measurements for the [Android's Activity](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation) transactions.
```

Comment:
Let's replace the word "jank" with something more meaningful. I've made a suggested edit to avoid this word, but if there's a better term, add it in.

```suggestion
Unresponsive UI and animation hitches annoy users and degrade the user experience. Two measurements to track these types of experiences are *slow frames* and *frozen frames*. If you want your app to run smoothly, you should try to avoid both. The SDK adds these two measurements for the [Android's Activity](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation) transactions.
```